website/source/docs/job-specification/task.html.md

```diff
@@ -50,7 +50,9 @@ job "docs" {
 - `kill_timeout` `(string: "5s")` - Specifies the duration to wait for an
   application to gracefully quit before force-killing. Nomad sends an `SIGINT`.
   If the task does not exit before the configured timeout, `SIGKILL` is sent to
-  the task.
+  the task. Note that the value set here is capped at the value set for
```

Comment:
Can you add a link: https://www.nomadproject.io/docs/agent/configuration/client.html#max_kill_timeout